server/src/contact/contact.service.ts

```diff
@@ -47,20 +48,18 @@ export class ContactService {
    * @return     {object}             Object representing a CRM contact
    */
   public async findOneByEmail(email: string) {
-    let results = null;
-    const activeCode = 1;
-
     try {
-      results = await this.crmService.get(`contacts?$filter=emailaddress1%20eq%20'${email}'%20and%20statuscode%20eq%20${activeCode}&$top=1`);
+      const { records: [firstRecord] } = await this.crmService.get('contacts', `
+        $filter=emailaddress1 eq '${email}'
+          and statuscode eq ${ACTIVE_CODE}
+        &$top=1
+      `);
+
+      return firstRecord;
     } catch(e) {
       const errorMessage = `Error finding contact by email. ${e.message}`;
       console.log(errorMessage);
       throw new HttpException(errorMessage, HttpStatus.UNAUTHORIZED);
     }
```

Comment:
We should maybe also move the Contact Serializer out of app.controller.ts and into contact.service.ts, for consistency with project.service.ts

Reply:
Yes!! Great catch thanks @godfreyyeung . Will do in follow up PR.